pennathletics/athletes.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function #PYTHON 3 BABY
+import models, parse
```

Comment:
Please use something like `pyflakes` to lint your code. PEP8 is the universal python code style guide, only 1 import statement per line.